app/graphql/types/query_type.rb

```diff
@@ -9,6 +9,14 @@ def current_account
     User.current
   end
 
+  field :current_profile, Types::Profile, null: true do
+    description 'Kitsu account details. You must supply an Authorization token in header.'
```

Comment:
Could you change the wording here?

```suggestion
    description 'Your Kitsu profile. You must supply an Authorization token in header.'
```